packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -52,21 +52,33 @@ type Props = {
   inverse?: boolean;
   unit?: string;
   max?: number;
+  getColor?: GetColor;
 };
 
+type GetColorArgs = {
+  palette: BackstageTheme['palette'];
+  value: number;
+  inverse?: boolean;
+  max?: number;
+};
+
+export type GetColor = (
```

Comment:
We should make sure this and the args type are exported as well, and would like to tweak the naming a little bit. Thinking `GaugePropsGetColor` and `GaugePropsGetColorOptions`